css/layout.css

```diff
@@ -18,6 +24,37 @@
   width: 100%;
 }
 
+header {
+  height: 3rem;
+  display: flex;
+  align-items: center;
+}
+
+main {
+  margin: 0;
+}
+
+img {
+  width: 100%;
+  object-fit: cover;
+  height: 33vh;
+}
+
+label {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 0;
+}
+
+label.column-full {
+  margin: 0;
+}
+
+.submit {
+  margin: 1.5rem 0;
+  text-align: right;
+}
+
```

Comment:
These should be moved over to `styles.css` unless you wanted to turn some of these into utility classes and impact the layout that way. `layout.css` should only have css applied to your container / row / col, or any utility classes. So for instance, you can create a `.margin-bottom` class and give it `margin-bottom: 1.5rem`, and give this class to multiple elements